src/main/kotlin/racing/view/InputView.kt

```diff
@@ -0,0 +1,22 @@
+package racing.view
+
+object InputView {
+
+    fun inputNumOfCars(): Int {
+        println("자동차 대수는 몇 대인가요?")
+
+        val numOfCars = readLine()!!.toIntOrNull() ?: throw IllegalArgumentException("자동차 대수는 숫자값으로 입력해주세요.")
```

Comment:
!!를 사용하시면 null이 나온 시점에 NPE가 날거라 Elvis operator가 의미가 없지 않을까 생각되네요. 아래와 같이 하면 어떨까요?

```suggestion
        val numOfCars = readLine()?.toIntOrNull() ?: throw IllegalArgumentException("자동차 대수는 숫자값으로 입력해주세요.")
```

Reply:
https://edu.nextstep.camp/s/56sRKVkV/ls/nCXdKx9a 여기에 있는 힌트만 보고 별생각 없이 작성하였습니다. ㅎㅎ 
말씀주신것처럼  safe call을 하는것이 더 나은 방법 같습니다. 감사합니다!! 🤗

[beec321](https://github.com/next-step/kotlin-racingcar/pull/893/commits/beec321a55b14c97f62c2bf1aefea6635381b4e4) 에 반영하였습니다.